index.html

```diff
@@ -26,6 +26,7 @@
         <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
         <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+        <script src="https://use.fontawesome.com/40afd82a34.js"></script>
```

Comment:
Use `https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css` instead of `https://use.fontawesome.com/40afd82a34.js`